Sources/Firebase/API/Endpoints.swift

```diff
@@ -3,4 +3,16 @@ import Foundation
 internal let APIBase = "https://firebasedynamiclinks.googleapis.com/"
 internal let APIVersion = "v1/"
 
-internal enum FirebaseAPIEndpoint { }
+internal enum FirebaseAPIEndpoint {
+
+    internal enum DynamicLinks {
+        // MARK: - Short Links
+        case shortLinks
+
+        var endpoint: String {
+            switch self {
+            case .shortLinks: return APIBase + APIVersion + "shortLinks"
```

Comment:
Good idea : fc16c47

Reply:
You can also remove the switch like in my example above I think. If later on, a new case require another kind of endpoint format, we'll reintroduce it but right now, it's totally useless.

Reply:
If we add a new endpoints for dynamic links, the code will compile but we do not define the route path in endpoint. It's better to keep the switch to show a compilation error if we add a new  endpoint. 
I know the DynamicLinks has only one endpoint and it will probably not others endpoints in the future but in this case, the enum is useless. The entire structure should be review. I think it's a good compromise to keep the switch for now.